scripts/docker-testnet/functions.sh

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env bash
+
+# Starts from 3, if the DOCKER_NETWORK_SUBNET ends with a 0. The first IP address is reserved for the gateway and the
+# second one is allocated to the seednode. Therefore the counting starts from 3. If you modify the DOCKER_NETWORK_SUBNET
+# variable, make sure to change this one accordingly too.
+IP_HOST_BYTE=3
+
+cloneRepositories() {
+  if [[ -n $CI_RUN ]]; then
+    echo "Repositories have been cloned in the CI"
+  else
+    cd $(dirname $MULTIVERSXDIR)
+
+    git clone git@github.com:multiversx/mx-chain-deploy-go.git || true
+    git clone git@github.com:multiversx/mx-chain-proxy-go.git || true
+  fi
+}
+
+createDockerNetwork() {
+  docker network create -d bridge --subnet=${DOCKER_NETWORK_SUBNET} ${DOCKER_NETWORK_NAME}
```

Comment:
maybe check if `DOCKER_NETWOR_NAME` already exists

Reply:
added `set -u`. Unbound variables will throw an error now.

Reply:
here I was referring to the fact that if the network is already created, it will fail because it is not able to create docker network again